### PR TITLE
Filter out keys first

### PR DESCRIPTION
**Pull request by goofansu**
Filter out keys first then no need to use `String.to_existing_atom`, because the keys are exactly what we need.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,11 +80,11 @@ defmodule GitHub do
...
-    :following_url, :gists_url, :starred_url, :subscriptions_url,
-    :organizations_url, :repos_url, :events_url, :received_events_url, :type,
-    :site_admin, :name, :company, :blog, :location, :email, :hireable, :bio,
-    :public_repos, :public_gists, :followers, :following, :created_at, :updated_at
+    "login", "id", "avatar_url", "gravatar_id", "url", "html_url", "followers_url",
+    "following_url", "gists_url", "starred_url", "subscriptions_url",
+    "organizations_url", "repos_url", "events_url", "received_events_url", "type",
+    "site_admin", "name", "company", "blog", "location", "email", "hireable", "bio",
+    "public_repos", "public_gists", "followers", "following", "created_at", "updated_at"
   ]
```
Comment (edgurgel) [inline README.md]: Could you change to use the word lists sigil? eg. `~w(login id avatar_url)`

**Comment by edgurgel** [comment]
Thanks! :+1:

**Comment by goofansu** [comment]
updated

**Comment by edgurgel** [comment]
Cheers!

**Pull request merged**